src/components/questionStructure/Task/variants/HandwritingTask/index.tsx

```diff
@@ -0,0 +1,64 @@
+import { Pencil2Icon } from '@radix-ui/react-icons'
+import { Button, Card, Dialog, Flex, Text } from '@radix-ui/themes'
+import { MathJax } from 'better-react-mathjax'
+import { FC } from 'react'
+import { useParams } from 'react-router-dom'
+
+import { DEFAULT_TEST_USERNAME } from '../../../../../utils/globalConstants'
+import { TaskType } from '../../constants'
+import { TaskBaseProps } from '../../types'
+import HandwritingEditor from './HandwritingEditor'
+import { ViewOnlyCanvas } from './ViewOnlyCanvas'
+import './index.scss'
+
+export interface HandwritingTaskProps extends TaskBaseProps<string> {
+  type: TaskType.PROCESSED_HANDWRITING
+  questionText: string
+}
+
+export const HandwritingTask: FC<HandwritingTaskProps> = ({
+  answer,
+  onAnswerUpdate,
+  disabled = false,
+  questionText,
+}) => {
+  const { username = DEFAULT_TEST_USERNAME } = useParams()
+
+  return (
+    <Dialog.Root>
+      <Dialog.Trigger>
+        <ViewOnlyCanvas initialData={{}} />
```

Comment:
What's the point of `initialData` if it's a hardcoded empty object? If it's used somewhere else, may be worth just using `{}` as default value in `ViewOnlyCanvas` for the prop.

Reply:
This was a placeholder for use in #21 when we added handwriting storage